task/git-clone/0.1/git-clone.yaml

```diff
@@ -190,6 +195,42 @@ spec:
       fi
       printf "%s" "${RESULT_SHA}" > "$(results.commit.path)"
       printf "%s" "${PARAM_URL}" > "$(results.url.path)"
+  - name: symlink-check
+    image: registry.redhat.io/ubi8:8.8-1009
+    env:
+    - name: PARAM_ENABLE_SYMLINK_CHECK
+      value: $(params.enableSymlinkCheck)
+    - name: PARAM_SUBDIRECTORY
+      value: $(params.subdirectory)
+    - name: WORKSPACE_OUTPUT_PATH
+      value: $(workspaces.output.path)
+    resources: {}
+    securityContext:
+      runAsUser: 0
```

Comment:
nit: this is not needed since ubi is using root(uid 0) by default.

Reply:
changed